pytket/extensions/qiskit/backends/aer.py

```diff
@@ -228,6 +233,17 @@ def process_circuits(
         if valid_check:
             self._check_all_circuits(circuits)
 
+        if hasattr(self, "crosstalk_params") and self.crosstalk_params is not None:
+            gate_times = get_gate_times_from_backendinfo(self.backend_info)
```

Comment:
I would be interested in discussing if this is the best way to obtain gate times. I would be okay with gate times being a property of `CrosstalkParams` to add user flexibility. I can't quite tell how easy it would be to build a purely hypothetical device with this set up (would I have to build a full set of characterisation information?).